nncf/common/pruning/utils.py

```diff
@@ -187,6 +192,93 @@ def get_previous_conv(graph: NNCFGraph, nncf_node: NNCFNode,
     return None
 
 
+def get_original_node_name(node_name: str):
+    return node_name.split('^')[0]
+
+
+def get_conv_in_out_channels(graph: NNCFGraph):
+    in_channels, out_channels = {}, {}
+    for node in graph.get_all_nodes():
+        if isinstance(node.module_attributes, ConvolutionModuleAttributes):
+            name = node.ia_op_exec_context.scope_in_model if hasattr(node, 'ia_op_exec_context') \
+                else get_original_node_name(node.node_name)
+            if name in in_channels and name in out_channels:
+                continue
+            in_channels[name] = node.module_attributes.in_channels
+            out_channels[name] = node.module_attributes.out_channels
+    return in_channels, out_channels
+
+
+def get_cluster_next_nodes(graph: NNCFGraph, pruned_groups_info,
+                           prunable_types: List[str]) -> Dict[int, List[str]]:
+    next_nodes = {}
+    for cluster in pruned_groups_info.get_all_clusters():
+        next_nodes_cluster = set()
+        cluster_nodes = set()
+        for cluster_node in cluster.nodes:
+            nncf_cluster_node = graph.get_node_by_id(cluster_node.nncf_node_id)
+            nncf_cluster_node_scope = nncf_cluster_node.ia_op_exec_context.scope_in_model \
+                if hasattr(nncf_cluster_node, 'ia_op_exec_context') \
+                else get_original_node_name(nncf_cluster_node.node_name)
+            cluster_nodes.add(nncf_cluster_node_scope)
+            curr_next_nodes = get_next_nodes_of_types(graph, nncf_cluster_node, prunable_types)
+
+            next_nodes_idxs = [n.ia_op_exec_context.scope_in_model if hasattr(n, 'ia_op_exec_context')
+                               else get_original_node_name(n.node_name) for n in curr_next_nodes]
+            next_nodes_cluster = next_nodes_cluster.union(next_nodes_idxs)
+        next_nodes[cluster.id] = list(next_nodes_cluster - cluster_nodes)
+    return next_nodes
+
+
+def count_flops(graph: NNCFGraph, input_shapes: dict, output_shapes: dict,
```

Comment:
Please, rename this function as `count_flops_in_nodes` or something like this more clear.